client/src/app/auth.guard.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { ConnectionService } from './connection.service';
+
+@Injectable({ providedIn: 'root' })
+export class AuthGuard implements CanActivate {
+    constructor(private router: Router, private connectionService: ConnectionService) { }
+
+    canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot) {
+        console.log('OnlyLoggedInUsers');
```

Comment:
you can remove the console.log() so when Dale works on or debugs his part. He might not know where that console.log come from too. Just to avoid confusion.